### DISCUSSION TITLE
Can we able to use google maps API for autocomplete field in Directus CMS?

### DISCUSSION THREAD

**Discussion by Decodeuprefo** [directus/directus#13823] [Archive]
Can we able to use google maps API for autocomplete field in Directus CMS?

![image](https://user-images.githubusercontent.com/83722746/173009804-e0a23eb9-78e6-4d0a-8164-946b78c589b6.png)

**Comment by joselcvarela** [comment]
From my tests I saw that Places API does not have CORS headers, leading to a CORS error.
I think you will need to use a proxy server or a custom interface using Google Javascript APIs.
But other members of the team may have other answers.

**Reply by Decodeuprefo** [reply]
I have already try to use this but not seems to work for me so I want details documentation in this and also example if possible
